src/styles/components/_usage-guidelines.scss

```diff
@@ -1,55 +1,59 @@
-$breakpoint-tablet: 1220px;
+$breakpoint-tablet: 860px;
 
 .cmp-usage-guidelines {
-  background-color: #01f5ac;
-  padding-left: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  line-height: 1.875rem;
```

Comment:
It looks like this `<p>` is inheriting line-height from here, you might need a separate selector to target the paragraph, for instance maybe something like `.cmp-usage-guidelines p`

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/54716846/200097123-fcce467e-a297-4bf0-b9e2-d744e9344173.png">

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/54716846/200097159-4cecce84-28e9-461a-91af-c6b9d4dd60ba.png">